src/common/com/intellij/plugins/haxe/ide/highlight/HXMLColorSettingsPage.java

```diff
@@ -33,6 +33,9 @@
   private static final AttributesDescriptor[] DESCRIPTORS = new AttributesDescriptor[]{
     new AttributesDescriptor("Key", HXMLSyntaxHighlighter.KEY),
     new AttributesDescriptor("Value", HXMLSyntaxHighlighter.VALUE),
+    new AttributesDescriptor("Comment", HXMLSyntaxHighlighter.COMMENT),
+    new AttributesDescriptor("Include", HXMLSyntaxHighlighter.INCLUDE),
```

Comment:
it's reference to inlined hxml file for haxe compiler: first line here -> ( https://github.com/andyli/minimatchx/blob/master/build.hxml )